geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -648,10 +656,9 @@ public int compareScores(double score1, double score2) {
 
     public static final int ORDERED_SET_ENTRY_OVERHEAD = memoryOverhead(OrderedSetEntry.class);
 
-    public OrderedSetEntry(byte[] member, byte[] score) {
+    public OrderedSetEntry(byte[] member, double score) {
       this.member = member;
-      this.scoreBytes = stripTrailingZeroFromDouble(score);
```

Comment:
With the removal of this line, the `stripTrailingZeroFromDouble()` method is not used anywhere outside of tests, so it can be removed.